vpr/src/place/initial_placement.cpp

```diff
@@ -161,9 +189,9 @@ static int try_place_macro(int itype, int ipos, t_pl_macro pl_macro) {
     return (macro_placed);
 }
 
-static void initial_placement_pl_macros(int macros_max_num_tries, int* free_locations) {
+static void initial_placement_pl_macros(int macros_max_num_tries, std::vector<std::vector<int>> free_locations) {
```

Comment:
free_locations should be passed by reference to avoid copying (and match previous behaviour).

Reply:
Done